server/app/mutations/grid_services/common.rb

```diff
@@ -352,7 +352,7 @@ def common_validations
               integer :initial_delay, default: 10
             end
           end
-          string :stop_signal
+          string :stop_signal, matches: /^((SIG)([A-Z0-9]+|RTMIN\+\d+|RTMAX-\d+)|\d+)$/i
```

Comment:
See #2324 for the hilarious reasons for why these regexp must use `/\A...\z/` instead of `/^...$`